src/content/workshops/functions/1-what-is-serverless/_index.md

```diff
@@ -4,12 +4,109 @@ weight: 101
 outputs: ["Reveal"]
 ---
 
-# Slide 1
+## What is Serverless?
 
-Test
+- Execution model
+- Pay as you consume
+- No need to control scaling
+- Event Driven
 
+{{% note %}}
+Serverless is an execution model. Everytime you hear the word serverless, it's talking about this cloud focused execution model. Running the same code locally is a different concept.
```

Comment:
```suggestion
Serverless is an execution model. When you hear the word "serverless", it's talking about a cloud-focused execution model. Running the same code locally is a different concept.
```
Do you really mean locally here? You can run functions locally for development and testing. I think you more mean running the same logic on your own application server.... or something like that, right?

Reply:
I mean as during the course of development. Not for production of course. Merging your changes.